src/main/java/org/testng/xml/XmlTest.java

```diff
@@ -582,21 +585,18 @@ public String toXml(String indent) {
   @Override
   public String toString() {
 //    return toXml("");
-    StringBuffer result = new StringBuffer("[Test: \"" + m_name + "\"")
-      .append(" verbose:" + m_verbose);
+    StringBuilder result = new StringBuilder("[Test: \"" + m_name + "\"").append(" verbose:").append(m_verbose);
```

Comment:
You missed `new StringBuilder().append("[Test: \"").append(m_name).append("\"")`